### PR TITLE
Fix broken links in readme.md

### PR DESCRIPTION
**Pull request by Hosuke**
### Update!
Please build spells in the proper [subproject](../dbt_subprojects/) directory. For more information, please see the main [readme](../README.md), which also links to a GH discussion with the option to ask questions.

### Contribution type
Please check the type of contribution this pull request is for:

- [ ] New spell(s)
- [ ] Adding to existing spell lineage
- [x] Bug fix

**Note:** You can safely discard any section below which doesn't apply based on selection above

---

### For new spell(s)
If you are building new spell(s), please provide the following information:

- **Spell name(s):** *schema.table_name*
- **Description:** [Detailed description of the new spell(s) and their purpose]
- **Who are the new spell(s) for?** [Internal team or general community]
- **How will the new spell(s) be used downstream?** [Description of downstream usage]
- **Implementation details:** [Information on how the spell(s) are implemented]
- **Test instructions:** [How to test the new spell(s)]
- **Related issue(s):** [Link to related issues, if any]

---

### For adding to existing spell lineage
If you are adding to an existing spell lineage, please provide the following information:

- **Description:** [Description of the changes made]

---

### For bug fixes
If you are fixing a bug, please provide the following information:

- **Description:** [Description of the bug fix]
- **Steps to reproduce:** [How to reproduce the bug]
- **Implementation details:** [Information on how the bug was fixed]
- **Test instructions:** [How to test the fix]
- **Related issue(s):** [Link to related issues, if any]

---

### Additional information
Please provide any additional information that might help us review your pull request:

- [Any additional information]

---

Thank you for your contribution!

### CODE REVIEW

File: dbt_subprojects/dex/models/trades/readme.md [hunk 1]
```diff
@@ -162,7 +162,7 @@ Example config block:
 
 ## Dependency on dex.info
 
-It's crucial to add a corresponding entry in [`dex.info`](/models/dex/dex_info.sql) when adding a new `dex`.
+It's crucial to add a corresponding entry in [`dex.info`](dbt_subprojects/dex/models/dex_info.sql) when adding a new `dex`.
```
Comment (Hosuke) [inline dbt_subprojects/dex/models/trades/readme.md:165]:
```suggestion
It's crucial to add a corresponding entry in [`dex.info`](/dbt_subprojects/dex/models/dex_info.sql) when adding a new `dex`.
```

File: dbt_subprojects/dex/models/trades/readme.md [hunk 2]
```diff
@@ -176,7 +176,7 @@ Seed tests are vital for ensuring that the output of our models aligns with the
 
 ### Define the Seed Schema
 
-Start by defining the schema of your seed in the [seeds/\_sector/dex/\_schema.yml](/seeds/_sector/dex/_schema.yml) file.
+Start by defining the schema of your seed in the [dbt_subprojects/dex/seeds/trades/\_schema.yml](dbt_subprojects/dex/seeds/trades/_schema.yml) file.
```
Comment (Hosuke) [inline dbt_subprojects/dex/models/trades/readme.md:179]:
```suggestion
Start by defining the schema of your seed in the [dbt_subprojects/dex/seeds/trades/\_schema.yml](/dbt_subprojects/dex/seeds/trades/_schema.yml) file.
```

File: dbt_subprojects/dex/models/trades/readme.md [hunk 3]
```diff
@@ -58,8 +58,8 @@ Adoption of the `base_` prefix for table aliases and `uniswap_` for macro names.
 
 ### Directory Structure
 
-- Macros/Models: `macros/models/_sector/dex/`
-- Platforms: `models/_sector/dex/trades/<blockchain>/platforms/`
+- Macros/Models: `dbt_subprojects/dex/macros/`
+- Platforms: `dbt_subprojects/dex/models/trades/<blockchain>/platforms`
```
Comment (Hosuke) [inline dbt_subprojects/dex/models/trades/readme.md:62]:
```suggestion
- Platforms: `dbt_subprojects/dex/models/trades/<blockchain>/platforms/`
```

File: dbt_subprojects/dex/models/trades/readme.md [hunk 4]
```diff
@@ -198,7 +198,7 @@ eg.
 
 ### Add the Corresponding Seed File
 
-Create and add the seed file in the [seeds/\_sector/dex/](/seeds/_sector/dex) directory. This file should contain the trades that verified on the blockchain explorers.
+Create and add the seed file in the [dbt_subprojects/dex/seeds/trades](/dbt_subprojects/dex/seeds/trades) directory. This file should contain the trades that verified on the blockchain explorers.
```
Comment (Hosuke) [inline dbt_subprojects/dex/models/trades/readme.md:201]:
```suggestion
Create and add the seed file in the [dbt_subprojects/dex/seeds/trades/](/dbt_subprojects/dex/seeds/trades) directory. This file should contain the trades that verified on the blockchain explorers.
```

**Comment by Hosuke** [comment]
[Readme preview](https://github.com/duneanalytics/spellbook/blob/befbddab4b46f91c9eb2919a33debab13d1a3b24/dbt_subprojects/dex/models/trades/readme.md)

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**